packages/webview_flutter/webview_flutter_platform_interface/lib/src/types/http_response_error.dart

```diff
@@ -4,6 +4,9 @@
 
 import 'package:flutter/foundation.dart';
 
+import '../../webview_flutter_platform_interface.dart';
```

Comment:
I think this could be used instead: 

```suggestion
import 'load_request_params.dart';
```